package.json

```diff
@@ -11,6 +11,8 @@
   },
   "devDependencies": {
     "@code-pushup/cli": "^0.50.0",
+    "@code-pushup/models": "^0.50.0",
+    "@code-pushup/utils": "^0.50.0",
```

Comment:
This is a left over from a failed/cancelled `cli-e2e-original:e2e-original`. See also in package-lock.json